### ISSUE TITLE
matte_floodfill doesn't work with ImageMagick 7

### ISSUE THREAD

**Issue by jcrate** [rmagick/rmagick#1296]
### Description
Image instance method matte_floodfill doesn't work with ImageMagick 7.

### Steps to Reproduce
```
require 'rubygems'
require 'bundler/setup'
require 'rmagick'

include Magick

plaque = Image.read('plaque_mask/name_bar_plaque.png').first

mask = plaque.channel(AlphaChannel)
mask.write('plaque_mask/mask1.png')

mask.fuzz = '75%'
mask.alpha(OpaqueAlphaChannel)
mask = mask.color_floodfill(mask.columns / 2, mask.rows / 2, Pixel.from_color('blue'))
mask.write('plaque_mask/mask2.png')

# turn on matte, make center area transparent
mask = mask.matte_floodfill(mask.columns / 2, mask.rows / 2)
mask.write('plaque_mask/mask3.png')

`open ./plaque_mask/`
```

### System Configuration
- ImageMagick version:  7.1.0-5 matte_floodfill does nothing; 6.9 works properly.
- RMagick version: 4.2.2
- Ruby version: 2.5.8
- Environment (Operating system, version and so on): macOS 11.6
- Additional information:

### Source Image
![name_bar_plaque](https://user-images.githubusercontent.com/53685/135577871-6a5f6bc0-dd21-4e90-b732-c8e6c5440bdb.png)

### IM6 mask3.png (center area is transparent)
![mask3_im6](https://user-images.githubusercontent.com/53685/135578129-f7424be1-5505-46af-8c43-22408ea129af.png)

### IM7 mask3.png (center area is not transparent)
![mask3_im7](https://user-images.githubusercontent.com/53685/135578142-1d6aebcd-fedf-4536-9023-617d857b146c.png)

**Comment by jcrate** [comment]
I haven't checked the rmagick code for this method yet but since the mask black/white is swapped between IM6 and IM7, I'd guess that matte_floodfill just isn't using the correct transparency for IM7.

**Comment by mockdeep** [comment]
@jcrate that would be very helpful. I'm not sure if we'll get a chance to get it fixed any time soon, but that'll be a step in the right direction.

**Comment by mockdeep** [comment]
Nevermind, fixed in #1299! Thanks to @dlemstra.

**Comment by jcrate** [comment]
Thanks @dlemstra for the quick fix!
